server/monitor.ts

```diff
@@ -274,8 +276,39 @@ export const Monitor = {
 		return bytes;
 	},
 
+	async version() {
+		function sh(command: string, path: string): Promise<[number, string, string]> {
```

Comment:
Maybe pull out this function?

Reply:
Into like `Monitor.sh` at the very least.

Reply:
Then I'd suggest making its signature ``sh(command, options)``, since the ``options`` passed to ``exec`` (including the current ``path``) are the same in all of its calls,